### ISSUE TITLE
Absolute position custom Portal for tooltip, menu, popover

### ISSUE THREAD

**Issue by gauthierrodaro** [radix-ui/primitives#913]
## Feature request

I am not sure if this is a bug or a feature request but it relates to https://github.com/radix-ui/primitives/issues/760

### Overview

Making absolute "custom" `Portal` works with element that are allowing it (e.g. tooltip, menu, popover,...). 

The following code only works if the "custom" `Portal` is position at the top left of the view screen (e.g. fixed)

```js
  <Tooltip.Root>
    <Tooltip.Trigger />
    <Portal.Root containerRef={portalRef}>
      <Tooltip.Content portalled={false}>
       Content
      </Tooltip.Content>
    </Portal.Root>
  </Tooltip.Root>
```

![CleanShot 2021-10-13 at 12 43 47](https://user-images.githubusercontent.com/6961490/137118592-481c6c37-e613-45a5-9265-756ce9e3fef2.gif)

The sandbox to show the problem is [here](https://codesandbox.io/s/fragrant-currying-m60nd?file=/App.js:1322-1631).

### Examples in other libraries

https://chakra-ui.com/docs/overlay/popover#rendering-the-popover-in-a-portal
https://chakra-ui.com/docs/components/portal

### Who does this impact? Who is this for?

This is a little bit more advance since it requires a custom Portal but I think all user would benefit from this.

### Additional context

Thank you for the great work on Radix UI! Let me know if you need help.

**Comment by jjenzz** [comment]
Yep! That's the one! Shall we close this as a duplicate of that? It seems once that is fixed the example in this issue should work?

**Comment by jjenzz** [comment]
Closing as duplicate of #370
